enterprise_access/apps/api_client/braze_client.py

```diff
@@ -0,0 +1,30 @@
+"""
```

Comment:
I thought there was a library we could use for this?

Reply:
we do. this is the [edx-braze-client](https://github.com/edx/braze-client) but this is the expected way to set up its keys, etc. tis taken from license-manager where we do the same

Reply:
👍 Ignore this one.